cmd/crc-embedder/cmd/embed.go

```diff
@@ -7,14 +7,12 @@ import (
 	"path"
 	"runtime"
 
+	"github.com/YourFin/binappend"
 	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/logging"
-	"github.com/code-ready/crc/pkg/download"
-
 	"github.com/code-ready/crc/pkg/crc/machine/hyperkit"
 	"github.com/code-ready/crc/pkg/crc/machine/libvirt"
-
-	"github.com/YourFin/binappend"
+	"github.com/code-ready/crc/pkg/download"
 	"github.com/spf13/cobra"
```

Comment:
I don't think this needs to be reordered?

Reply:
I removed this change